apps/fz_http/lib/fz_http_web/controllers/auth_controller.ex

```diff
@@ -116,12 +103,30 @@ defmodule FzHttpWeb.AuthController do
     end
   end
 
-  # This can be called if the user attempts to visit one of the callback redirect URLs
-  # directly.
-  def callback(conn, params) do
-    conn
-    |> put_flash(:error, inspect(params) <> inspect(conn.assigns))
-    |> redirect(to: ~p"/")
+  def saml_callback(conn, _params) do
+    key = {idp, _} = get_session(conn, "samly_assertion_key")
```

Comment:
Just now noticing this as well -- this will crash with a 500 if a user visits this route directly because of a MatchError (`samly_assertion_key` won't be set). Think we should fix in this PR or save for another?